kuzzleio/kuzzle
# Use the newly published hot reloader Ergol

Closes #1953

## What does this PR do ?

Extract the reloader from the kuzzle repo and publish it in order to be able to use it from any repository.
The reloader is here to watch project files when developing and restart the project when there is some modification.

### How should this be manually tested?

Clone and run kuzzle, pick a random file in the lib directory, add anything and watch your terminal: the project should restart.

### Other changes

The deleted reloader file must be located somewhere else... Go check the brand new repository [ergol](https://github.com/kuzzleio/ergol) and don't hesitate to use it in your other projects!
